packages/next/styles/src/__snapshots__/StyleSheet.test.js.snap

```diff
@@ -0,0 +1,46 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`converts a css string into an object 1`] = `
+Object {
+  "alert": Object {
+    "__className": "pf-c-alert",
+    "__inject": [Function],
+  },
+  "button": Object {
+    "__className": "pf-c-button",
+    "__inject": [Function],
+  },
+  "modifiers": Object {},
+  "vars": Object {},
+}
+`;
+
+exports[`places custom properties in the modifers object 1`] = `
+Object {
+  "button": Object {
+    "__className": "pf-c-button",
+    "__inject": [Function],
+  },
+  "modifiers": Object {},
+  "vars": Object {
+    "c_button__PaddingTop": "--pf-c-button--PaddingTop",
+    "global__primary_color__100": "--pf-global--primary-color--100",
```

Comment:
like we discussed... eventually it might be nice to also extract the variable value for consumption in JS as well (I can see this being useful in Canvas/Charts/etc. in the future...). I know the typical use case would be to define these in CSS though. I don't see this as a blocker for moving this forward at all, but a follow issue might be nice?

Reply:
Yeah. That would be a future thing.  The downside being that we have no way of knowing the value at runtime unless we force a,`<ThemeProvider />` type of deal and manage those in React instead.  I would be up for that if people are interested.